src/loaders/common.c

```diff
@@ -364,3 +364,58 @@ void libxmp_set_type(struct module_data *m, const char *fmt, ...)
 	vsnprintf(m->mod.type, XMP_NAME_SIZE, fmt, ap);
 	va_end(ap);
 }
+
+static int schism_tracker_epoch(void)
```

Comment:
If you are anyway going to put this in its own function rather than just keeping the epoch as a magic number as before, I would suggest to make this function take parameters year, month and day. Then it can be re-used to check against readable Schism Tracker versions when toggling tracker quirk flags (xmp doesn't currently do this but e.g. OpenMPT does it)

Reply:
Yes, I think that would be a better solution. I've updated this function to take a date instead in case it ever needs to be reused (worst case it should optimize exactly the same as before).